nima/webclient/webclient/src/test/java/io/helidon/nima/webclient/HttpClientTest.java

```diff
@@ -113,6 +113,16 @@ public FakeHttpClientRequest fragment(String fragment) {
             return this;
         }
 
+        @Override
+        public FakeHttpClientRequest followRedirects(boolean followRedirects) {
+            return null;
```

Comment:
I think it should `return this` rather than `return null`

Reply:
ehm... yes... thank you.